cli/src/commands/diff.js

```diff
@@ -0,0 +1,87 @@
+const fs = require('fs');
+const minimist = require('minimist');
+const { exec } = require('child_process');
+
+const help = ui => {
+    const { bold, cyan, dim } = ui.colors;
+
+    ui.print(`
+${bold('NAME')}
+        comfy diff - Show diff of two configurations for a given environment
+
+${bold('SYNOPSIS')}
+        ${bold('comfy')} diff <environment> <hash|tag> [<hash|tag>]
+
+${bold('OPTIONS')}
+        <environment>   Name of the environment (must already exist in project)
+        <hash>          Hash or tag of your configuration to diff it
```

Comment:
<hash|tag> just upper
```suggestion
        <hash|tag>          Hash or tag of your configuration to diff it
```